packages/auth/lib/challenges/AuthorizeTokenChallenge.js

```diff
@@ -0,0 +1,55 @@
+const { newAuthError } = require('../AuthError')
+
+const AccessTokenMissingError = newAuthError(
+  'authorize-token-challenge-access-token-missing',
+  'api/auth/authorizeToken/accessTokenMissing'
+)
+const TokensExceededError = newAuthError(
+  'authorize-token-challenge-tokens-exceeded',
+  'api/auth/authorizeToken/tokensExceeded'
+)
+
+const MAX_VALID_TOKENS = 1
```

Comment:
add a comment that this must no be changed, because otherwise the TTL constraint can't be guaranteed.